db/migrate/20230215131812_create_arts.rb

```diff
@@ -0,0 +1,15 @@
+class CreateArts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :arts do |t|
+      t.string :title
+      t.text :description
+      t.integer :price
```

Comment:
price should be a float.
Changing that'll require some changes in the model validation too

Reply:
done!